orttraining/orttraining/python/orttraining_pybind_state.cc

```diff
@@ -142,13 +142,13 @@ TrainingConfigurationResult ConfigureSessionForTraining(
   std::cout << "[ConfigureSessionForTraining] pipeline_config" << std::endl;
   training::TrainingSession::TrainingConfiguration::PipelineConfiguration pipeline_config;
   pipeline_config.do_partition = true;
-  // The following cut may fail because a non-differentiable path is cut. 
-  //training::TrainingSession::TrainingConfiguration::CutEdge cut_edge0("208");
-  //training::TrainingSession::TrainingConfiguration::CutEdge cut_edge1("426");
-  //training::TrainingSession::TrainingConfiguration::CutEdge cut_edge2("89", {"501"});
-  //training::TrainingSession::TrainingConfiguration::CutInfo cut_info{cut_edge0, cut_edge1, cut_edge2};
-  training::TrainingSession::TrainingConfiguration::CutEdge cut_edge0("534");
-  training::TrainingSession::TrainingConfiguration::CutInfo cut_info{cut_edge0};
+  // The following cut may fail because a non-differentiable path is cut.
+  training::TrainingSession::TrainingConfiguration::CutEdge cut_edge0("208");
+  training::TrainingSession::TrainingConfiguration::CutEdge cut_edge1("426");
+  training::TrainingSession::TrainingConfiguration::CutEdge cut_edge2("89", {"501"}, true);
+  training::TrainingSession::TrainingConfiguration::CutInfo cut_info{cut_edge0, cut_edge1, cut_edge2};
+  // training::TrainingSession::TrainingConfiguration::CutEdge cut_edge0("534");
+  // training::TrainingSession::TrainingConfiguration::CutInfo cut_info{cut_edge0};
```

Comment:
I think this section is cleaned up in Wei-Sheng's 0.5 API PR, and if we merge this PR after #5815 then we won't need to update orttraining_pybind_state (and hence won't require the front-end team to review here).